src/Host/Client/Impl/Extensions/RStringExtensions.cs

```diff
@@ -91,5 +91,13 @@ public static class RStringExtensions {
             }
             return s;
         }
+
+        public static string ToRPath(this string s) {
```

Comment:
It's still the full path in those cases, it's doesn't return paths with `~` in them.